packages/trader/src/App/Containers/CashierModal/PaymentAgent/payment-agent-withdraw.jsx

```diff
@@ -0,0 +1,209 @@
+import classNames           from 'classnames';
+import PropTypes            from 'prop-types';
+import React                from 'react';
+import {
+    Button,
+    Dropdown,
+    Form,
+    Input }                 from 'deriv-components';
+import { getDecimalPlaces } from '_common/base/currency_base';
+import Localize             from 'App/Components/Elements/localize.jsx';
+// import RadioGroup           from 'App/Components/Form/Radio';
+import { localize }         from 'App/i18n';
+import { connect }          from 'Stores/connect';
+import {
+    validNumber,
+    getPreBuildDVRs }       from 'Utils/Validator/declarative-validation-rules';
+import PaymentAgentReceipt  from './payment-agent-receipt.jsx';
+import Loading              from '../../../../templates/_common/components/loading.jsx';
+
+const validateWithdrawal = (values, { balance, currency, payment_agent }) => {
+    const errors = {};
+
+    // if (values.payment_method === 'payment_agent' && (!values.payment_agent || !/^[A-Za-z]+[0-9]+$/.test(values.payment_agent))) {
```

Comment:
I know why it is, but could we add a comment for why this is disabled? Or perhaps we can have a tagged TODO for this and the commented code starting from line 95.